src/components/Main/types.ts

```diff
@@ -56,8 +56,10 @@ export enum SCOPE_CHANGE_EVENTS {
   RECENT_ACTIVITY_SPAN_LINK_CLICKED = "RECENT_ACTIVITY_SPAN_LINK_CLICKED",
   IDE_CODE_LENS_CLICKED = "IDE/CODE_LENS_CLICKED",
   IDE_NOTIFICATION_LINK_CLICKED = "IDE/NOTIFICATION_LINK_CLICKED",
+  IDE_CUSTOM_PROTOCOL_LINK_CLICKED = "IDE/CUSTOM_PROTOCOL_LINK_CLICKED",
```

Comment:
Can we extract all IDE events on level upper?

Reply:
SCOPE_CHANGE events are currently related to `Main` app only